### PR TITLE
initial commit with notebook and requirements.txt

### PR DESCRIPTION
**Pull request by deep-diver**
@sayakpaul 

Is there a way to build Gradio app for HuggingFace Space when the package should be built from source?

### CODE REVIEW

**Comment by sayakpaul** [comment]
> Is there a way to build Gradio app for HuggingFace Space when the package should be built from source?

Couldn't find one. I guess we can settle with the PT model for now for the Space. Meanwhile, we could ask a question on the HF discussion forum.

File: notebooks/TFSegFormer.ipynb [hunk 1]
```diff
@@ -0,0 +1,322 @@
+{
```
Comment (sayakpaul) [inline notebooks/TFSegFormer.ipynb:1]: <p>If this notebook is supposed to run on Colab then I guess we need to show the pip installation from source (specifying the <code>tf-segformer</code> branch).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (sayakpaul) [inline notebooks/TFSegFormer.ipynb:1]: <p>Why is there there bullet point beside TensorFlow? 🧐</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (sayakpaul) [inline notebooks/TFSegFormer.ipynb:1]: <p>We can do a <code>print(logits.shape)</code> and add a comment on the shape of the logits: <code>(batch_size, num_labels, height, width)</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (sayakpaul) [inline notebooks/TFSegFormer.ipynb:1]: <p>Nevermind it's already here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

**Comment by sayakpaul** [comment]
Looks good with minor comments. I will do some investigations tomorrow to see why the PT outputs differ a bit from the TF outputs.

File: notebooks/TFSegFormer.ipynb (hunk 1 above)
Comment (deep-diver) [reply notebooks/TFSegFormer.ipynb:1]: <p>my bad. that is just my habit, I will remove the bullet points</p>

Comment (deep-diver) [reply notebooks/TFSegFormer.ipynb:1]: <p>I have included instructions in README of this PR. But I will try to run this on colab as well</p>

**Comment by deep-diver** [comment]
@sayakpaul 

I think it is done now! I will add Colab badge after this PR is merged
